src/bin/nydusd/fs_cache.rs

```diff
@@ -250,7 +250,15 @@ impl FsCacheHandler {
             .write(true)
             .read(true)
             .create(false)
-            .open(path)?;
+            .open(path)
+            .map_err(|e| {
+                error!(
+                    "Failed to open cachefiles device {}. {}. Please try `modprobe cachefiles`",
```

Comment:
"Failed to open cachefiles device {}:" is better. :)

Reply:
Yeah, I think `modprobe cachefiles` is no needed to be seen here, there might be other reasons.

Reply:
@hsiangkao   You are right. I face this issue after cachefiles enabled. Do you know what are the reasons cause the cachefiles device issue?